### ISSUE TITLE
关于梯度累积的实现

### ISSUE THREAD

**Issue by macheng6** [OpenRLHF/OpenRLHF#753]
求问，openrlhf中如果想实现梯度累积怎么实现呢？
- 手动配置deepspeed的config文件
- loss = loss / accum_steps,   model.backward(loss), model.step()
但是第二种方法，deepspeed中去掉了optimizer.zero_grad()，不清楚是否还有梯度累积的作用。

**Comment by hijkzzz** [comment]
"deepspeed中去掉了optimizer.zero_grad()" 这个有相关文档吗?

**Comment by macheng6** [comment]
> "deepspeed中去掉了optimizer.zero_grad()" 这个有相关文档吗?

![Image](https://github.com/user-attachments/assets/4841a026-1722-4f3b-b516-bd60eb8df93f)
https://github.com/deepspeedai/DeepSpeedExamples/tree/master/training/HelloDeepSpeed
